src/verifier/SourcifyVerifier.ts

```diff
@@ -83,9 +92,10 @@ export class SourcifyVerifier extends AbstractVerifier implements Verifier {
       logObject(this.logger, 'debug', postQueries, 2);
       return await axios.post(SOURCIFY_API_URL, postQueries);
     } catch (error: any) {
-      this.logger.debug(error.message);
-      this.logger.debug(error.response.data.message);
-      throw new Error(`Failed to connect to Sourcify API at url ${SOURCIFY_API_URL}`);
+      this.logger.info(error.message);
+      this.logger.info(`Response from ${SOURCIFY_API_URL}:`);
+      this.logger.info(error.response.data);
```

Comment:
I'm thinking these should stay `debug` messages?

Reply:
Added some context into the PR message. I think we should return these messages to the user why the verification failed?